packages/graphiql/test/e2e-server.js

```diff
@@ -24,8 +24,14 @@ app.use(express.json());
 app.post('/graphql', handler);
 app.get('/graphql', handler);
 
-app.use(express.static(path.resolve(__dirname, '../')));
-app.use('index.html', express.static(path.resolve(__dirname, '../dev.html')));
+// On CI we test the UMD build
+if (process.env.CI === 'true') {
+  app.use(express.static(path.join(__dirname, '..')));
+} else {
+  app.get('/', (req, res) => {
+    res.redirect('http://localhost:5173');
```

Comment:
this is fine as long as it's serving the unminified umd version

Reply:
I made to use umd only on CI, and you can see that all tests pass

locally I replace umd import with https://github.com/graphql/graphiql/blob/ad75cdaacd0f0043dce436d6869299af0a618094/packages/graphiql/vite.config.mts#L102-L111